toolchains/solidity/core/crates/linter-lib/src/linter.rs

```diff
@@ -60,6 +64,14 @@ impl SolidLinter {
         Ok(())
     }
 
+    pub fn initialize_ignore_file(&mut self, filepath: &str) -> Result<(), SolidHunterError> {
```

Comment:
What if there is nested `.solidhunterignore` ? 
Example : on at the root of the project and one in the `./lib/specific-folder` folder ? 
All the `.solidhunterignore` files should be found recursively in this function